### PR TITLE
North West 4 - Morteza Khojasteh - JavaScript - Week 1 (Mandatory)

### PR DESCRIPTION
**Pull request by Morteza-Khojasteh**
**Volunteers: Are you marking this coursework?** _You can find a guide on how to mark this coursework in `HOW_TO_MARK.md` in the root of this repository_

# Your Details

- Your Name:
- Your City:
- Your Slack Name:

# Homework Details

- Module:
- Week:

# Notes

- What did you find easy?

- What did you find hard?

- What do you still not understand?

- Any other notes?

### CODE REVIEW

File: mandatory/3-function-output.js [hunk 1]
```diff
@@ -5,12 +5,15 @@ function getRandomNumber() {
```
Comment (lannem) [inline mandatory/3-function-output.js:5]: can you explain what the first 2 functions do?

**Review by lannem — COMMENTED** [review]
Nice work :)

**Comment by Morteza-Khojasteh** [comment]
> Nice work :)

Thanks

File: mandatory/3-function-output.js (hunk 1 above)
Comment (Morteza-Khojasteh) [reply mandatory/3-function-output.js:5]: I have a function **getRandomNumber()** without parameter inside the function there is another function The **Math. random() function** returns a floating-point, pseudo-random number in the range 0 to less than 1 (inclusive of 0, but not 1) with approximately uniform distribution over that range — which you can then scale to your desired range
// expected output: a number from 0 to <10

**Pull request closed without merging**